apis/Google.Cloud.Diagnostics.AspNetCore/Google.Cloud.Diagnostics.AspNetCore.IntegrationTests/Logging/LoggingTest.cs

```diff
@@ -185,6 +185,24 @@ public class LoggingTest
                 Assert.Equal(3, buildResources.Count());
             }
         }
+
+        [Fact]
+        public async Task Logging_Scope()
+        {
+            string testId = Utils.GetTestId();
+            DateTime startTime = DateTime.UtcNow;
+
+            var builder = new WebHostBuilder().UseStartup<NoBufferResourceLoggerTestApplication>();
+            using (TestServer server = new TestServer(builder))
+            using (var client = server.CreateClient())
+            {
+                await client.GetAsync($"/Main/Scope/{testId}");
+                var results = _polling.GetEntries(startTime, testId, 1);
+                Assert.Single(results);
```

Comment:
I don't think this is necessary: `results.Single()` will throw below if there isn't a single result.